faucet/src/http.rs

```diff
@@ -262,7 +289,7 @@ pub async fn start_http(
     allowance_config: AllowanceConfig,
 ) -> jsonrpc_http_server::CloseHandle {
     let mut io = IoHandler::default();
-    let store = Store::new(Config::new("./kv")).expect("Unable to open kv store");
+    let store = Store::new(Config::new("./kv").flush_every_ms(100)).expect("Unable to open kv store");
```

Comment:
I think the faucet previously did not write state to disk so the previous claim check did persist restarts. Picked the value arbitrarily but we can probably increase it to something more sensible.